application/Profile/LocalProfile.py

```
@@ -126,8 +140,9 @@
             # need to handle any write permission issues, once observed
             raise e
 
-    def get_session_info(self, session_id):
+    def get_session_info(self, session_id) -> tuple[object, object, object, None, object]:
```

Ruff (0.8.2)

Missing type annotation for function argument `session_id`

(ANN001)